src/summary/profile/identifiers-card.component.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import HorizontalLabelValue from "../cards/horizontal-label-value.component";
+import { match } from "react-router";
+
+export default function IdentifiersCard(props: IdentifiersCardProps) {
+  const labelStyles = {
+    color: "var(--omrs-color-ink-medium-contrast)"
+  };
+
+  const valueStyles = {
+    color: "var(--omrs-color-ink-high-contrast)",
+    fontWeight: 500
+  };
+
+  return (
+    <SummaryCard name="Identifiers" match={props.match}>
+      {props.currentPatient &&
+        props.currentPatient.identifier.map(id => (
+          <SummaryCardRow linkTo="/" key={id.system}>
+            <SummaryCardRowContent>
+              <HorizontalLabelValue
```

Comment:
I think it's important to make sure the preferred identifier is the first one in the list.